Documentation/ProjectFile/prj/processes/process/RICHARDS_FLOW/material_property/porous_medium/porous_medium/a_id.md

```diff
@@ -0,0 +1,2 @@
+It specifies the material ID of a porous medium. The material IDs of elements
+should be given in the mesh's cell data array "MaterialIDs".
```

Comment:
If the data array 'MaterialIDs' does not exist in the mesh then ogs crashs. For this reason, i would write instead of 'should' either 'must' or 'have'.